### ISSUE TITLE
Alphabetical Tags

### ISSUE THREAD

**Issue by Auroratide** [Auroratide/sifetti#16]
Tags should be sorted alphabetically by default.

**Comment by Auroratide** [comment]
Oops I forgot it was already sorted.
